needle/modules/configuration/mdm_assess.py

```diff
@@ -0,0 +1,130 @@
+from core.framework.module import BaseModule
+from core.framework.framework import FrameworkException
+from core.device.device import Device
+from core.utils.utils import Utils
+from core.utils.constants import Constants
+import re, sys, plistlib
+
+class Module(BaseModule):
+    meta = {
+        'name': 'MDM Assess',
+        'author': 'Oliver Simonnet (@MWRLabs)',
+        'description': 'Automated MDM Configuration Assessment Module.',
+        'options': (
+            ('template', True, True, 'Configuration template.[Plist]'),
+            ('mode', False, True, 'Output mode [1|2|3].')
```

Comment:
as requested in both previous reviews, please remove this verbosity mode and stick to the default levels offered by the framework